app/views/grids/_thumbnail.html.erb

```diff
@@ -1,5 +1,8 @@
   <div class="image-container row">
     <% nodes.each_with_index do |node, index| %>
+        <% if index % 3 == 0 && index > 0 %>
+          <div class="clearfix"></div>
```

Comment:
Ah ok but i think the extra "visible-____" classes are important so it breaks only when it's a the width where 3 cols are shown, does that make sense? Check out the example i pasted into my comment, or the Bootstrap docs I linked to -- should be a small addition here. I think `visible-lg` maybe?

Reply:
and THANKS!!!! 🎉